gemfirexd/tools/src/main/java/com/pivotal/gemfirexd/internal/impl/tools/ij/utilMain.java

```diff
@@ -438,7 +438,7 @@ private int runScriptGuts() {
 
 					ijResult result = ijParser.ijStatement();
 					endTime = displayResult(out,result,connEnv[currCE].getConnection(),
-					    beginTime /* GemStoneAddition */);
+					    beginTime /* GemStoneAddition */, true);
```

Comment:
Can't this get the first token from ijParser? See ijStatement in ij.jj. It does a lookahead to get the first token, so that can also check one of update/delete/insert/put and set some flag in parser to indicate that.